docs/source/information_model.rst

```diff
@@ -76,6 +76,28 @@ coordinate. Reference sequences used to describe Sequence Locations should be ve
      - 1..1
      - A coordinate representing the end of a genomic location.
 
+.. _reading-frame:
+
+Reading Frame
+#############
+A common attribute of a gene fusion is whether the reading frame is preserved in the expressed gene
```

Comment:
We should reword "expressed gene product" to  "transcript" in alignment with other manuscript changes.